### PR TITLE
net-misc/httpie: Bump to version 2.2.0

### PR DESCRIPTION
**Pull request by rseichter**
Ebuild unchanged from previous version. A second commits removes obsolete ebuilds and patch files.

### CODE REVIEW

File: net-misc/httpie/httpie-2.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+inherit bash-completion-r1 distutils-r1
+
+DESCRIPTION="Modern command line HTTP client"
+HOMEPAGE="https://httpie.org/ https://pypi.org/project/httpie/"
+SRC_URI="https://github.com/jakubroztocil/httpie/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```
Comment (thesamesam) [inline net-misc/httpie/httpie-2.2.0.ebuild]: IUSE/RESTRICT are handled by distutils_enable_tests, so we can drop them

File: net-misc/httpie/httpie-2.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/pygments[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.22.0[${PYTHON_USEDEP}]"
+DEPEND="test? (
+		${RDEPEND}
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/pyopenssl[${PYTHON_USEDEP}]
+		dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
```
Comment (thesamesam) [inline net-misc/httpie/httpie-2.2.0.ebuild]: We can remove both the pytest line and the ${RDEPEND} inclusion in test? ( ... ) which are handled by distutils_enable_tests

File: net-misc/httpie/httpie-2.2.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (thesamesam) [inline net-misc/httpie/httpie-2.2.0.ebuild:7]: Can you add Python 3.9 if tests pass?

File: net-misc/httpie/httpie-2.2.0.ebuild (hunk 1 above)
Comment (rseichter) [reply net-misc/httpie/httpie-2.2.0.ebuild]: Stripping the ebuild of unnecessary lines seems a good idea; thanks for the suggestions.

File: net-misc/httpie/httpie-2.2.0.ebuild (hunk 3 above)
Comment (rseichter) [reply net-misc/httpie/httpie-2.2.0.ebuild:7]: Upstream docs state that Python >= 3.6 should work, but I have not yet tested it. It is however on my to-do list.

**Pull request closed without merging**